lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
```

Comment:
The server sent event spec doesn't explain this really well, but the Last-Event-Id is required to be passed on as a header : http://www.w3.org/TR/2011/WD-eventsource-20110310/#refsRFC3864

Current impl will not be enable the browser to auto-resume the stream in case it drops off